sql/migrations/schema-postgresql-move-to-mirrors.sql

```diff
@@ -0,0 +1,98 @@
+DROP TABLE IF EXISTS filemetadata, mirrors CASCADE;
+DROP MATERIALIZED VIEW IF EXISTS filemetadata_mirror_count;
+
+CREATE TABLE filemetadata
+(
+    id integer GENERATED ALWAYS AS IDENTITY,
+    path character varying NOT NULL,
+    path_hash bytea GENERATED ALWAYS AS (digest((path)::text, 'sha256'::text)) STORED,
+    mtime timestamp with time zone,
+    size bigint,
```

Comment:
I think we should stick to having hashes in the separate table, the way it was in old schema. 
E.g. hashes may require a lot of additional space, especially zsums can grow really huge. That will impact number records per page and can practically slow down fetching rows and scanning the table. Also having separate table can simplify maintenance, backup and recovery. Additionally then we can easier keep NOT NULL for most of required metadata.

Reply:
maybe only split out the bittorrent, pgp and zsync hashes? the rest should be fixed length and we could even define that length in the schema.

Reply:
I don't know. I'd prefer to stick to old code, but probably one table can be manageble; we can split it later as well :)